examples/knowledge_tuning/translation_example/nllb_server.py

```
@@ -0,0 +1,102 @@
+                    "index": 0,
+                    "logprobs": None,
+                    "finish_reason": "stop",
+                }
+            ],
+        }
+    except Exception as e:
+        return {"error": f"Translation failed: {str(e)}"}, 500
+
+
+@app.get("/v1/completions")
+async def translate(source_lang: str, target_lang: str, prompt: str):
```

Pylint (3.3.7)

[error] function already defined line 23

(E0102)